docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,111 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+Validation and Verification Criteria
+------------------------------------
 
-   all of this
+Obejective
+++++++++++
+External
+ - Model results should be checked against local TB epidemiology
+    - e.g. rate of decline in burden of disease should be compared
+      with historical evidence.
+ - Compare our results (e.g. ICERs) to similar models or empirical
+   assessments where possible
+Internal
+ - Test hypotheses without develop a full simulation. 
```

Comment:
by this, do you mean a paired-down sim? or a back-of-the-envelope calculation?

Reply:
The bullet points in external part are things we can check based on non-GBD evidences. Back envelope calculation is the step before post-processing raw sim outputs that tells whether your results fit into a reasonable magnitude.